### PR TITLE
mgr/dashboard: Fix typo in pools management

### PR DESCRIPTION
**Pull request by LenzGr**
Typo fix: "suite your needs" -> "suit your needs"

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/src/locale/messages.xlf [hunk 1]
```diff
@@ -3455,7 +3455,7 @@
         </context-group>
       </trans-unit><trans-unit id="f7a13e4807afa6a9eda3db57a0a2a110dbae8ad7" datatype="html">
         <source><x id="ICU" equiv-text="{mode, select, edit {...} other {...}}"/> the pool
-          mirror peer attributes for pool <x id="START_TAG_KBD" ctype="x-kbd" equiv-text="&lt;kbd&gt;"/><x id="INTERPOLATION" equiv-text="{{ poolName }}"/><x id="CLOSE_TAG_KBD" ctype="x-kbd" equiv-text="&lt;/kbd&gt;"/> and click 
+          mirror peer attributes for pool <x id="START_TAG_KBD" ctype="x-kbd" equiv-text="&lt;kbd&gt;"/><x id="INTERPOLATION" equiv-text="{{ poolName }}"/><x id="CLOSE_TAG_KBD" ctype="x-kbd" equiv-text="&lt;/kbd&gt;"/> and click
```
Comment (epuertat) [inline src/pybind/mgr/dashboard/frontend/src/locale/messages.xlf]: Just out of curiosity, is this change manual or some editor has removed that trailing whitespace?

Comment (LenzGr) [reply src/pybind/mgr/dashboard/frontend/src/locale/messages.xlf]: Gah, didn't notice that. Must have been my editor (MS Visual Studio Code)...
Hmm. Let me try to fix this.

Comment (LenzGr) [reply src/pybind/mgr/dashboard/frontend/src/locale/messages.xlf]: Should be fixed now.

Comment (epuertat) [reply src/pybind/mgr/dashboard/frontend/src/locale/messages.xlf]: +1

Comment (LenzGr) [reply src/pybind/mgr/dashboard/frontend/src/locale/messages.xlf]: Meh, manually changing these strings didn't work, I'm getting `Please run 'npm run i18n' and commit the modified 'messages.xlf' file.`. Redoing this once again...

**Pull request merged**